api/src/main/java/com/debugmafia/clueless/controller/StatusController.java

```diff
@@ -0,0 +1,22 @@
+package com.debugmafia.clueless.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.debugmafia.clueless.service.GameService;
+
+@RestController
+public class StatusController {
+
+  @Autowired
+  private GameService gameService;
+
+  @GetMapping(path = "/api/status", produces = "application/json;charset=UTF-8") 
+  @ResponseBody
+  public Boolean getStatus() throws Exception {
```

Comment:
This needs to return a `Map<String, Boolean>`. Also, there is no need for `@ResponseBody` here, that's implied because this is a `@RestController`. Same thing goes for the `produces = "application/json;charset=UTF-8"` property, that's implied with being a rest controller.

The returned map can have a single property, `gameIsInProgress`, with the value set to `gameService.getGameIsInProgress()`

Reply:
I see what you're saying, but doesn't it make sense to have it the same as we do in Lobby controller, for consistency sake? That's what i was going for.

Reply:
I'm fine either way, I was just trying to make everything look the same code wise (even though its doing the same thing under the hood). I don't feel strongly one way or the other: I just wanted to share my train of thought.